aiohttp/client.py

```diff
@@ -167,6 +173,74 @@ def request(self, method, url, *,
 
         return resp
 
+    @asyncio.coroutine
+    def ws_connect(self, url, *,
+                   protocols=(),
+                   timeout=10.0,
+                   ws_response_class=None,
```

Comment:
The only possible other change I would make to this patch would be to move ws_response_class to the constructor function to be consistent with `ClientSession.request` method.